seed/migrations/0110_auto_20200323_1509.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
@macintoshpie think you could shift this migration to the end and delete `seed/migrations/0111_merge_20200324_0837.py`?